keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,162 @@
+
+## Motivation
+
+SIG Docs publishes Kubernetes documentation on kubernetes.io, in line with its [charter](https://github.com/kubernetes/community/blob/master/sig-docs/charter.md#scope), and also sets standards for website content. Prior to this KEP there were no clear guidelines or standards showing what documentation is acceptable.
+
+The Kubernetes documentation is currently a mix of both 1) documentation describing the Kubernetes open source project; and 2) content describing how to install or use Kubernetes on several third party Kubernetes offerings.
+
+Some third party content is necessary in order for Kubernetes to function. For example, Docker, networking policy (CNI plugins), Ingress controllers, and logging all require third party components.
+Pages like [Logging Using Elasticsearch and Kibana](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/) are highly specific to a third party offering and seem more like third party product documentation than Kubernetes open source documentation.
+
+The goal of this KEP is to reach and document a consensus on what types of third party content are appropriate for inclusion in Kubernetes documentation; standards for including third-party content; and to drive a consistent policy toward third party content inclusion.
+This KEP focuses on the following issues related to third party content inclusion:
+
+1. What third party content is appropriate for inclusion in the Kubernetes documentation?
+1. Does third party content in sections such as [Getting Started](https://kubernetes.io/docs/setup/) in the docs provide sufficient value to the reader that they should remain?
+1. Is there a list of content pages that are so focused on third party product usage that they should be removed or updated from the Kubernetes documentation?
+1. When should the Kubernetes documentation host third party documents that we are not the source/authority of?
+1. How does the Kubernetes project handle third party content that is not kept up to date or hosts?
+1. Can feature owners flag when third party content is *required*, as opposed to preferable or common?
+1. Who decides when to include third-party content?
+1. What standard of quality and review must be met before docs include third-party content?
+1. To what extent should SIG Docs advocate for third-party content providers to host their own content, or decline to host third-party content altogether?
+
+### Goals
+
+- Policy on including content about third party products in Kubernetes web documentation.
+- Published guidelines for contributors and reviewers that clarify requirements for third party content and allow documentation reviewers to reject advertising.
+
+
+### Non-Goals
+
+- Outright removal of all content relating to vendors and projects outside the CNCF ecosystem.
+
+## Proposal
+
+TBD. Detailed policies will come out of the discussion around this KEP.
```

Comment:
IMO just a simple decision flowchart should be sufficient to outline the process.

Reply:
Let's hope so!

Reply:
> IMO just a simple decision flowchart should be sufficient to outline the process.

Agreed, that's an ideal outcome of this KEP.